templates/single.html

```diff
@@ -55,4 +51,4 @@
 </main>
```

Comment:
Before the closing `main`, there's a closing group tag/comment missing:

```
</div>
<!-- /wp:group -->
```

Reply:
Interesting. This must have been missing for some time. I've added the closing Group tag here.

Reply:
Great catch!